src/mscorlib/src/System/Threading/Tasks/Task.cs

```diff
@@ -6011,8 +6011,7 @@ public static Task<Task> WhenAny(params Task[] tasks)
 
             // Make a defensive copy, as the user may manipulate the tasks array
             // after we return but before the WhenAny asynchronously completes.
-            int taskCount = tasks.Length;
-            Task[] tasksCopy = new Task[taskCount];
+            Task[] tasksCopy = new Task[tasks.Length];
             for (int i = 0; i < taskCount; i++)
```

Comment:
Yeah compiled it by changing that aswell, only to realize now that would do more harm in the end. My bad!